### DISCUSSION TITLE
[feat] Read later in sidebar

### DISCUSSION THREAD

**Discussion by ilovethensa** [Floorp-Projects/Floorp#668] [Feature Request]
I propose the implementation of a comprehensive "Read Later" menu directly within the sidebar, offering users a centralized hub for organizing and accessing saved content. Users can expediently add links to this dedicated list by utilizing a context menu – simply right-click on the desired link and select "Add to Read Later." This holistic feature is designed to enhance user efficiency and provide a unified space for managing their reading queue.

**Comment by breadbeforerice** [comment]
If you set extensions.pocket.enabled to true in about:config, you can use pocket. T
It is probably close to what you are looking for

**Reply by tcelestino** [reply]
There is native Firefox feature, it would be interesting to have it in Floorp as well, wouldn't it
